test/functional/action-params.spec.ts

```diff
@@ -91,6 +96,29 @@ describe("action parameters", () => {
                 return `<html><body>${firstId},${secondId}</body></html>`;
             }
 
+            @Post("/session/")
+            @UseBefore(session({
+                secret: "19majkel94_helps_pleerock",
+            }))
+            addToSession(@Session() session: Express.Session) {
+                (session as any).testElement = "@Session test";
+                (session as any).fakeObject = {
```

Comment:
`fakeObject`? I added it just to be sure that the `@Session("testElement")` return the correct object, not the whole session container - the coverage report shows that the line 154 line `return request.session[param.name];` is not covered which is wrong because it's tested and works ;)